spanner/spanner_snippets/graph_snippet.go

```diff
@@ -0,0 +1,486 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Command spanner_snippets contains runnable snippet code for Cloud Spanner.
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"regexp"
+	"time"
+
+	"cloud.google.com/go/spanner"
+	"google.golang.org/api/iterator"
+
+	database "cloud.google.com/go/spanner/admin/database/apiv1"
+	adminpb "google.golang.org/genproto/googleapis/spanner/admin/database/v1"
+)
+
+type command func(ctx context.Context, w io.Writer, client *spanner.Client) error
+type adminCommand func(ctx context.Context, w io.Writer, adminClient *database.DatabaseAdminClient, database string) error
+
+var (
+	commands = map[string]command{
+		"insertdata":                    insert,
+		"insertdatawithdml":             insertWithDml,
+		"updatedatawithdml":             updateWithDml,
+		"updatedatawithgraphqueryindml": updateWithGraphQueryInDml,
+		"query":                         query,
+		"querywithparameter":            queryWithParameter,
+		"deletedata":                    delete,
+		"deletedatawithdml":             deleteWithDml,
+	}
+
+	adminCommands = map[string]adminCommand{
+		"createdatabase":              createDatabase,
+		"updateallowcommittimestamps": updateAllowCommitTimestamps,
+	}
+)
+
+// [START spanner_create_database]
```

Comment:
This sample already exists, I think:
https://github.com/GoogleCloudPlatform/golang-samples/blob/HEAD/spanner/spanner_snippets/spanner/spanner_create_database.go

Reply:
I created a distinct `spanner_create_database_with_property_graph`